index.html

```diff
@@ -4446,24 +4799,27 @@ <h3 id="Warrior">Warrior</h3>
             <tr>
                <td>2
                </td>
-               <td>Mobile
+               <td>Sundering Strike
                </td>
-               <td><strong>Move swiftly and without hindrance. </strong>
+               <td><strong>Utterly break an enemy's defences.</strong>
                   <p>
-                     Your base movement speed is increased by 10ft. You also ignore difficult
-                     terrain.
+                     Make a melee weapon attack against a creature and on a hit you may either:
```

Comment:
I understand the intent, but this just feels weird to me.  Metagaming "what is the AC" before the strike happens, plus the discouraging the teamplay option (reducing AC) for personal glory (more damage) doesn't sit overly well. 

To mitage the second one, maybe do bonus damage based on the AC reduction caused and merging the "or" into one?

Reply:
Yeah it was previously combined but I thought it was a bit too generically good, but yeah maybe it's ok to leave as is as bonus damage that's relatively easy to get for big enemies